src/components/CategoryList/styles.js

```diff
@@ -0,0 +1,112 @@
+import styled from 'styled-components';
+import { colors, sizes, media, fonts } from '../../helpers/styles/index';
```

Comment:
O index no final não é necessário
```suggestion
import { colors, sizes, media, fonts } from '../../helpers/styles';
```